### PR TITLE
chore: improve MixProject definitions and documentation

### PR DESCRIPTION
**Pull request by nelsonmestevao**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
+      licenses: ["MIT"],
+      links: %{"GitHub" => "https://github.com/filipecabaco/francis"}
+    ]
+  end
+
+  defp docs do
+    [
+      main: "readme",
+      extras: ["README.md"],
+      source_ref: "v#{@version}",
```
Comment (filipecabaco) [inline mix.exs:39]: unsure this will be the way 👀

Comment (nelsonmestevao) [reply mix.exs:39]: Should I just delete in the mean time? 👀

Comment (filipecabaco) [reply mix.exs:39]: for now yes, still need to set all the CI side of things at then will make that decision probably 😅

Comment (nelsonmestevao) [inline mix.exs:39]:
```suggestion
```

**Review by filipecabaco — APPROVED** [review]

**Pull request merged**